apps/dashboard/app/(app)/keys/[keyAuthId]/new/page.tsx

```diff
@@ -17,7 +18,7 @@ export default async function ApiPage(props: { params: { keyAuthId: string } })
   if (!keyAuth || keyAuth.workspace.tenantId !== tenantId) {
     return notFound();
   }
-
+  revalidate(`/keys/${props.params.keyAuthId}`);
```

Comment:
this revalidates on page render, but it should revalidate when a mutation finishes
you need to call this in the mutation